### PR TITLE
Issue #15456: Define violation messages for PackageAnnotationCheck

### PR DESCRIPTION
**Pull request by luvtyping**
Issue #15456 
Violation messages for PackageAnnotationCheck

### CODE REVIEW

File: src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/Example1.java [hunk 1]
```diff
@@ -9,7 +9,8 @@
 //non-compiled with javac: Compilable with Java7
 
 // xdoc section -- start
-@Deprecated // violation below
+@Deprecated
+// violation below, 'Package annotations must be in the package-info.java info.'
```
Comment (romani) [inline src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/Example1.java:13]: Please move violation comment under package to not split package and annotation

**Review by romani — CHANGES_REQUESTED** [review]
Same for other

File: src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/Example1.java (hunk 1 above)
Comment (luvtyping) [reply src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/Example1.java]: done

**Review by romani — APPROVED** [review]
Thanks a lot.

Appreciate future contribution

**Pull request merged**